src/components/projects/Inspector.tsx

```diff
@@ -110,7 +110,7 @@ const Inspector: React.FC<Props> = ({ editor }) => {
                   key="comments"
                 />
               </InspectorTabPanel>
-              {config.quarterback.enabled && (
```

Comment:
The check for the tab header also needs to change (L#95)

Reply:
Yes, thanks!